source/common/runtime/runtime_features.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/runtime/runtime_features.h"
+
+namespace Envoy {
+namespace Runtime {
+
+// Add additional features here to enable the new code paths by default.
+//
+// These features should not be overridden with run-time guards without a bug
```

Comment:
I think you are talking to end users here, right? Can you clarify slightly and say something like, "If issues are found that that require a feature to be disabled, it should be reported upstream ASAP, ..."